docs/src/manual/variables.md

```diff
@@ -1017,13 +1006,49 @@ julia> @constraint(model, x in SecondOrderCone())
 An alternate syntax to `x in Set` is to use the `set` keyword of
 [`@variable`](@ref). This is most useful when creating anonymous variables:
 ```julia
-x = @variable(model, [1:2, 1:2], set = PSDCone())
+x = @variable(model, [1:3], set = SecondOrderCone())
 ```
 
 !!! note
     You cannot delete the constraint associated with a variable constrained on
     creation.
 
+### Example: positive semidefinite variables
+
+Declare a matrix of JuMP variables to be positive semidefinite
```

Comment:
The reader might wonder what's the difference with the `@variable(model, x[1:2, 1:2], PSD)` above (same for symmetric)

Reply:
Added a comment.